### PR TITLE
Kg/postgres update

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

Thicker description of the Supermarket Upgrade. I added more specific information on handling the backup (move it off-system) and also an optional post-vacuum 'working' backup step. It can be nice not to roll all the way back to the archive copy. Of course, this assumes that the 'working' copy is, well, working.

Having an 'archive' copy is simply best practice. The 'working' copy comes from experience with technical services in libraries. I have seen things. 🚒 

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/supermarket/supermarket_upgrade.md [hunk 1]
```diff
@@ -13,70 +13,144 @@ publishDate = 2022-01-03
     weight = 25
 +++
 
+Chef Supermarket uses the PostgreSQL database. [PostgreSQL 9.6 is EOL](https://endoflife.date/postgresql) and Private Supermarket users should upgrade to [Supermarket 4.x](https://www.chef.io/downloads/tools/supermarket) and migrate to [Postgres 13](https://www.postgresql.org/about/news/postgresql-13-released-2077/).
```
Comment (tas50) [inline docs-chef-io/content/supermarket/supermarket_upgrade.md:16]: We're actually shipping 9.2 right now :(

File: docs-chef-io/content/supermarket/supermarket_upgrade.md [hunk 2]
```diff
@@ -13,70 +13,144 @@ publishDate = 2022-01-03
...
+    This will restart the `runsvdir`, `runsv`, and `svlogd` service processes that run Chef Supermarket.
+
+
+## Migrate to PostgreSQL 13
 
-## Upgrade Supermarket
+TODO: What package SEMVER has the PostgreSQL update
 
-Do we need to upgrade Supermarket first?
+PostgreSQL 13.3 is installed with the Supermarket 4.X.X upgrade package.
```
Comment (tas50) [inline docs-chef-io/content/supermarket/supermarket_upgrade.md:64]: It'll be 4.3+

File: docs-chef-io/content/supermarket/supermarket_upgrade.md (hunk 1 above)
Comment (RajeshPaul38) [reply docs-chef-io/content/supermarket/supermarket_upgrade.md:16]: @tas50 It's 9.3 actually.

Comment (tas50) [reply docs-chef-io/content/supermarket/supermarket_upgrade.md:16]: Oh yep 🤦

Comment (kagarmoe) [reply docs-chef-io/content/supermarket/supermarket_upgrade.md:16]: OK

**Comment by kagarmoe** [comment]
It looks like #2216 will replace this. Can we make sure @tas50 's comments are applied over there?

**Comment by saghoshprogress** [comment]
We can close this and #2216 will replace this one.

**Pull request closed without merging**